modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/ImportCGC.pm

```diff
@@ -164,24 +151,72 @@ sub run {
 sub write_output {
   my $self = shift;
 
-  $self->print_pipelogFH("Passing $source_info{source_name_short} import (".$self->required_param('species').") for checks (check_phenotypes)\n") if ($self->debug);
+  $self->print_pipelogFH("Passing $source_info{source_name_short} import (".$self->required_param('species').") for checks (check_phenotypes)\n") if ($debug);
   close($self->logFH) if defined $self->logFH ;
   close($self->errFH) if defined $self->errFH ;
-  close($self->pipelogFH) if defined $self->pipelogFH ;
+  close($self->pipelogFH) if defined $debug ;
 
   $self->dataflow_output_id($self->param('output_ids'), 2);
 
 }
 
+=head2 validate_input_file
+
+  Arg [1]    : string $infile
+               The input file name
+  Example    : $obj->validate_input_file($infile)
+  Description: Validates the format of the input file.
+  Returntype : Boolean
+  Exceptions : none
+
+=cut
+
+sub validate_input_file {
+  my ($self, $input_file) = @_;
+
+  my $valid = 1;
+
+  # empty file
+  if(-z $input_file) {
```

Comment:
Suggesting you change the $input_file to $self->workdir."/".$input_file

Reply:
Done. Thanks for noticing it.